Makefile

```diff
@@ -1,13 +1,14 @@
 # This is a convenience Makefile wrapping cmake calls
 # All targets should be defined in CMake
 
+build := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/build
 .PHONY: all
 all: build/CMakeCache.txt
-	ninja -C build/ symlinks
+	ninja -C ${build} symlinks
 
 .DEFAULT: build/CMakeCache.txt
-	ninja -C build/ $@
+	ninja -C ${build} $@
 
 build/CMakeCache.txt:
-	@mkdir -p build
-	@cmake -B build -S . -G Ninja
+	@mkdir -p ${build}
+	@cmake -B ${build} -S . -G Ninja
```

Comment:
Maybe this needs quoting now?

Reply:
You mean if spaces would be included? I have tested but only with space-less paths so far

Reply:
I tried and found http://savannah.gnu.org/bugs/?712 as well as many references explaining that the best is to avoid spaces in filenames when using make. So no quoting "needed" because it does not do what you expect nor fix problems in make (unlike bash)